devops/openshift/queue-consumers-deploy.yml

```diff
@@ -211,6 +211,23 @@ objects:
                 requests:
                   cpu: ${CPU_REQUEST}
                   memory: ${MEMORY_REQUEST}
+              readinessProbe:
+                failureThreshold: 3
+                httpGet:
+                  path: /health/
+                  port: "${{PORT}}"
+                initialDelaySeconds: 10
```

Comment:
Same for inital delay seconds.

Reply:
initial delay for applications are determined, to how long to wait before the container health checks needs to be done. For workers and queue consumers, i determined this value by doing tests from 1, 5, 10, 20 and 30. If we have more initial delay that will cause time delay to get the application to started state, if we have less initial delay the container will restart again and again, thinking the health check is failing. So we have to find the median.

Reply:
I got the initial delay for this file. But api-deploy.yml was having 40. I suspect we might have considered the migration run time initially when db migrations were running during deployment. 

It is outside the PR so we can consider that outside the ticket.